.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,14 +1,62 @@
 ---
 name: Bug Report
-about: Report 🐞 Bugs
-title: ''
+about: Create a 🐞 Bug Report to Improve Athens
+title: 'Descriptive Title with Category'
 labels: 'type: 🐞 bug'
 assignees: ''
+ID number: ''
+Category: ''
 
 ---
 
-**Problem**
+**Before Beginning:**
+
+- [ ] Read [Contribution Guidelines](https://app.gitbook.com/@athensresearch/s/handbook/~/drafts/-MZt8Vmd14cTrIp7xfV9/contributing-1/code-of-conduct/@archived) and [Code of Conduct](https://app.gitbook.com/@athensresearch/s/handbook/~/drafts/-MZt8Vmd14cTrIp7xfV9/contributing-1/code-of-conduct/@archived)
+
+- [ ] Check for similar [issues](https://github.com/athensresearch/athens/issues)
+- [ ]  Ask questions in [Discussions](https://github.com/athensresearch/athens/discussions)  *Bug reports ≠ questions!*
+- [ ] Get help in [Discord](https://discord.com/channels/708122962422792194/708126131160875079/814921829902057492)
+
+**Describe the Problem**
+
+Summarize the *one* issue in 1-3 sentences using keywords. One bug per report!
+
+**Steps to Reproduce**
+Below is an example of what the steps to reproduce a bug might look like:
+1. Go to '...'
+2. Log in '...'
+3. Click on '...'
+3. Scroll down to '...'
+4. Provide visuals: URLs, screenshots, demo, test case, repo links to clone, etc.
+
+**Expected Result**
+
+What did you expect to happen?
+
+**Actual Result**
+
+What happened? How many times? Try to reproduce 3x before submitting.
+
+**Additional Context**
+
+What other notes could help the community solve this bug?
+
+**Choose a Priority Level**
+- [ ] Critical: this bug makes the software unusable
+- [ ] High: the bug affects a main feature and there is no way around it
+- [ ] Medium: the bug affects a feature but there is a workaround
+- [ ] Low: minor visual issue, typo, etc.
+
+**Your Environment**
+
+Athens version:
+
+Operating system:
+
+Platform and Testing Environment:
+
+Packages:
+
+Dependencies:
```

Comment:
What would you expect users to put here?

Reply:
This area is to link to other actions or issues in Github in indicate that fixing the bug might be dependent on something else being done first. It is also be a reminder to mark the issue with a dependent label. 

It supports better end-to-end visibility.

Reply:
I meant the template should explain what should be put there. For example: `Issues this issue depends on:`